pkg/kubelet/status/manager_test.go

```diff
@@ -64,7 +65,8 @@ func (m *manager) testSyncBatch() {
 
 func newTestManager(kubeClient client.Interface) *manager {
 	podManager := kubepod.NewBasicPodManager(kubepod.NewFakeMirrorClient())
-	podManager.AddPod(testPod)
```

Comment:
Can we add a new function `getTestPod()` return a new test pod every time? This will be more fool proof than asking each test to make a copy.